js/ui/file_manager/ui.file_manager.file_uploader.js

```diff
@@ -0,0 +1,376 @@
+import $ from "../../core/renderer";
+import { extend } from "../../core/utils/extend";
+import eventsEngine from "../../events/core/events_engine";
+import { Deferred, when } from "../../core/utils/deferred";
+import Class from "../../core/class";
+
+import Widget from "../widget/ui.widget";
+import Button from "../button";
+import ProgressBar from "../progress_bar";
+import Popup from "../popup";
+
+const FILE_MANAGER_FILE_UPLOADER_CLASS = "dx-filemanager-fileuploader";
+const FILE_MANAGER_FILE_UPLOADER_FILE_INPUT_CLASS = FILE_MANAGER_FILE_UPLOADER_CLASS + "-fileinput";
+
+const FILE_MANAGER_PROGRESS_PANEL = "dx-filemanager-progresspanel";
+
+const FILE_MANAGER_PROGRESS_BOX = "dx-filemanager-progressbox";
+const FILE_MANAGER_PROGRESS_BOX_TITLE = FILE_MANAGER_PROGRESS_BOX + "-title";
+const FILE_MANAGER_PROGRESS_BOX_PROGRESS_BAR = FILE_MANAGER_PROGRESS_BOX + "-progressbar";
+const FILE_MANAGER_PROGRESS_BOX_CANCEL_BUTTON = FILE_MANAGER_PROGRESS_BOX + "-cancel-button";
+
+var FileManagerFileUploader = Widget.inherit({
+
+    _initMarkup: function() {
+        this._progressPanel = this._createComponent($("<div>"), FileManagerUploadProgressPanel, {});
+        this.$element().append(this._progressPanel.$element());
+
+        this._renderFileInput();
+
+        this.$element().addClass(FILE_MANAGER_FILE_UPLOADER_CLASS);
+
+        this.callBase();
+    },
+
+    _renderFileInput: function() {
+        this._$fileInput = $("<input>")
+            .attr("type", "file")
+            .prop({
+                multiple: "multiple",
+                tabIndex: -1
+            })
+            .addClass(FILE_MANAGER_FILE_UPLOADER_FILE_INPUT_CLASS);
+
+        eventsEngine.on(this._$fileInput, "change", this._onFileInputChange.bind(this));
+        eventsEngine.on(this._$fileInput, "click", e => {
+            e.stopPropagation();
+            return true;
+        });
+
+        this.$element().append(this._$fileInput);
+    },
+
+    _onFileInputChange: function() {
+        var files = this._$fileInput.prop("files");
+        if(files.length === 0) return;
```

Comment:
```suggestion
        if(files.length === 0) {
          return;
        }
```

Reply:
Check for the similar entries please

Reply:
Done. All similar constructions are fixed in code.